### PR TITLE
feat(contrib/ec2): add support for internal ELBs

### PR DESCRIPTION
**Pull request by carmstrong**
replaces #2864

### CODE REVIEW

**Comment by bacongobbler** [comment]
sweet. Nice optimization there. Code LGTM!

File: docs/installing_deis/aws.rst [hunk 1]
```diff
@@ -80,7 +80,7 @@ Customize cloudformation.json
 -----------------------------
 
 Any of the parameter defaults defined in deis.template.json can be overridden by setting the value
-in `cloudformation.json`_ like so:
+in `cloudformation.json`_ - for example, to configure all of the optional settings:
```
Comment (mboersma) [inline docs/installing_deis/aws.rst]: I think this should be an em-dash `—` or better, a new sentence starting "For example,".

**Comment by mboersma** [comment]
Code LGTM, my grammar comment is optional.

File: docs/installing_deis/aws.rst (hunk 1 above)
Comment (carmstrong) [inline docs/installing_deis/aws.rst]: I'll make this change and then merge.

**Pull request merged**